pkg/services/sqlstore/annotation_cleanup.go

```diff
@@ -0,0 +1,87 @@
+package sqlstore
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+// AnnotationCleanupService is responseible for cleaning old annotations.
+type AnnotationCleanupService struct {
+	batchSize int64
+	log       log.Logger
+}
+
+var (
+	alertAnnotationType     = "alert_id <> 0"
+	dashboardAnnotationType = "dashboard_id <> 0 AND alert_id = 0"
+	apiAnnotationType       = "alert_id = 0 AND dashboard_id = 0"
+)
```

Comment:
Does `const` work here? Would be preferable if it does work, since it would mark them as immutable:
```suggestion
const (
	alertAnnotationType     = "alert_id <> 0"
	dashboardAnnotationType = "dashboard_id <> 0 AND alert_id = 0"
	apiAnnotationType       = "alert_id = 0 AND dashboard_id = 0"
)
```